editor/componentdropable.dre

```diff
@@ -49,7 +49,7 @@ either express or implied. See the License for the specific language governing p
     
     // Determine if this is a tree drop or a canvas drop
     var isTreeDrop = droptarget.isA(dr.treeitem);
-    if (isTreeDrop) droptarget = droptarget.treecontainer.data.ref;
+    if (isTreeDrop) droptarget = droptarget.data.__ref;
```

Comment:
Note, the tree ignores any attributes with names that start with '__', so in order to keep the underlying Dreem object that's attached to the tree data to also be walked by the tree-diff, I renamed it to '__ref'.